pkg/kubelet/server/server.go

```diff
@@ -165,7 +165,9 @@ func (s *KubeletServer) Run(_ []string) error {
 	//
 	// TODO(erictune): convert all cloud provider scripts and Google Container Engine to
 	// use only --api_servers, then delete --etcd_servers flag and the resulting dead code.
-	if len(s.EtcdServerList) > 0 && len(s.APIServerList) > 0 {
+	if len(s.APIServerList) == 0 {
+		glog.Fatalf("--api_server is required.")
```

Comment:
hmm perhaps I'll just remove this bit as there appears to be semantics that are in flux.  I'm not a big fan of more flags at this point as long as it's handled.  

At some point it would be nice to distill the number of options down or provide .config files.